src/mainwindow.h

```diff
@@ -110,7 +110,7 @@ private slots:
     Ui::MainWindow *ui;
 
     QString path;
-    int _mode;
+    int _mode; // TODO init
```

Comment:
Its initialized:

```c++
    mode = new QComboBox(ui->Edit);
    mode->addItems(QStringList() << QLatin1String("Commonmark") << QLatin1String("GitHub"));
    mode->setCurrentIndex(1);
    _mode = 1;
```

Reply:
I mean, for a reader who has not written the code it might not be obvious and it may happen that one day you add a method that tries to read this variable before the value got set. So, imo, always better to initialize as early as possible

Reply:
Now I get it. Since it is always set to 1 by default, this should not be a problem.